### ISSUE TITLE
Loading a prologue savegame causes wrong damage calculation

### ISSUE THREAD

**Issue by drujd** [tton-bugs-hunters/tton-bug-tracker#7]
Save a game BEFORE selecting a doppleganger in the mirror. Exit the game, start it again and load the savegame. Proceed with the doppleganger selection, character creation etc. Once the first crisis with the Sorrow begins, Last Castoff always does only 1 point of damage, regardless of the amount of effort put into the attack. This problem occurs also later in Sagus Cliffs.

**Comment by drujd** [comment]
No, the issue does not just happen to Last Castoff in prologue. The problem just occurred to me with Aligern during the crisis with Quoro (after unequipping his mace). His fists just do 1 damage, even if you put max effort into the attack. The statistics also always show 1+0 on the attack effort selection "dialogue".

**Comment by drujd** [comment]
(Steam version, Windows 10 x64)

**Comment by drujd** [comment]
Worth of note is that Aligern's tattoo ability does add damage to his attacks. It is just effort that does not.

**Comment by vv221** [comment]
I’ve had the same bug on Linux with the GOG version, so I guess it might happen with all versions of the game.

**Comment by vv221** [comment]
Thread on inXile forums:
[Issues with fist attack](https://forums.inxile-entertainment.com/viewtopic.php?f=47&t=17392)

**Comment by drujd** [comment]
Yes. I think I know exactly when the problem manifests and how to work around it:
1. Unequip any character's weapon
2. Save
3. Load the game
4. Enter crisis - the problem occurs

5. Load thje game again
6. Equip and unequip any weapon
7. Enter crisis - effort works correctly
